tests/coalaTest.py

```diff
@@ -65,6 +65,17 @@ def test_did_nothing(self):
         self.assertIn('Nothing to do.', stderr)
         self.assertFalse(stdout)
 
+    def test_format(self):
+        retval, stdout, stderr = execute_coala(coala.main, 'coala', '--ci',
+                                               '--format', '-b',
+                                               'SpaceConsistencyTestBear',
+                                               '-f', '*.py'
+                                               )
+        self.assertEqual(retval[0], 0)
+        self.assertEqual(retval[1], 0)
+        self.assertIn('Executing section cli...', stdout)
+        self.assertTrue(stderr)
```

Comment:
If there is a content - 
validate it , either full validation or partially using `assertIn` or regex.
If its empty - 
simply assert it empty or false.

So, we need to validate contents of stderr.